kodilla-exception/src/main/java/com/kodilla/exception/challenge/FirstChallenge.java

```diff
@@ -0,0 +1,42 @@
+package com.kodilla.exception.challenge;
+
+import java.io.IOException;
+
+public class FirstChallenge {
+
+    public double divide(double a, double b) throws ArithmeticException {
+        if (b == 0) {
+            throw new ArithmeticException();
+        }
+        return a / b;
+    }
+
+    /**
+     * This main can throw an ArithmeticException!!!
+     *
+     * @param args
+     */
+    public static void main(String[] args) {
+
+        FirstChallenge firstChallenge = new FirstChallenge();
+
+        try {
+
+            double result = firstChallenge.divide(3, 0);
+
+            System.out.println(result);
+        } catch (NullPointerException e)
+        {
+            e.printStackTrace();
```

Comment:
Nie ma potrzeby przechwytywac tutaj NullPointerException, poniewaz jedyny przypadek kiedy to zadziala to gdy firstChallende == null. Nawet gdyby do tego doszlo, to wyjatek bedzie chwycony przez Exception e